### PR TITLE
CU-1mtgtrk - Minor changes was made. if creds.json is already existe…

### PR DESCRIPTION
**Pull request by gal-sh**
…d, it will not rewrited again.

### CODE REVIEW

**Comment by elonsalfati** [comment]
Task linked: [CU-1mtgtrk CLI minor changes](https://app.clickup.com/t/1mtgtrk)

**Comment by RavidEliyahu** [comment]
@ElonSalfati Please take a look at packages/cli/web/index.js file on this pr,
is this the logic you wanted?

File: packages/cli/src/commands/auth/login-ci.js [hunk 1]
```diff
@@ -1,33 +1,42 @@
...
 
 // login command execution
 class LoginCICommand extends BaseCommand {
   /**
    * Execute the login command
    */
-  async run() {ß
+  async run() {
     // generate random state validation
     const state = Math.floor(Math.random()*90000) + 10000;
```
Comment (elonsalfati) [inline packages/cli/src/commands/auth/login-ci.js]: Do something like this:

```javascript
async run() {
  let creds = require('~/.rb/cred.json')
  if (!creds?.refresh_token) {
    LoginCommand.run([ ... ])
  }

  creds = require('~/.rb/cred.json')
  console.log(creds?.refresh_token)
}
```

File: packages/cli/src/commands/auth/login-ci.js [hunk 2]
```diff
@@ -1,33 +1,42 @@
...
 
-    // open the redirect uri with the default browser
-    open(consolePath)
-
     // start spinner
     cli.ux.action.start("Waiting for authentication...")
+    // open the redirect uri with the default browser
+    open(consolePath);
+
+    process.on('exit', ( code ,signal) => {
```
Comment (elonsalfati) [inline packages/cli/src/commands/auth/login-ci.js]: Avoid using `process.on('exit')`. process.on('exit') is designed for graceful shutdowns (for example when you hit ctrl+c on a server and you need to clear some system processes) and not standard processes that can be a part of the program

File: packages/cli/web/index.js [hunk 3]
```diff
@@ -8,31 +8,52 @@ const mustacheExpress = require("mustache-express")
 const credsPath = path.join(os.homedir(), ".rb")
 
 /**
- * Routing handler helper
- * @param {boolean} ciServer indicates if the connection is throw a ci server
- * @param {object} req - request obj
- * @param {object} res - request res
- * @param {string} state - state number for validation propose
+ * Check if creds are existed , if not write them again
```
Comment (elonsalfati) [inline packages/cli/web/index.js:11]: If you use the normal LoginCommand - you should be able to remove any code that you added to the server. 

Or at least most of it.

**Review by elonsalfati — CHANGES_REQUESTED** [review]

**Comment by elonsalfati** [comment]
Check the unit tests

File: packages/cli/src/commands/auth/login-ci.js (hunk 2 above)
Comment (gal-sh) [reply packages/cli/src/commands/auth/login-ci.js]: got it, thanx

**Review by elonsalfati — APPROVED** [review]

**Pull request merged**